### ISSUE TITLE
Lang property for document contents

### ISSUE THREAD

**Issue by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#795]
The interface of I-analyzer is in English, but the contents of documents are often not. To avoid screen readers pronouncing the contents of all documents as English text, we should set the [`lang`  attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)  to the appropriate setting.

Setting the property in HTML is easy enough, the potentially tricky part is how we can best encode this in the corpus definition.

The easiest version is to set a `language` property for corpora, e.g. `language = 'dutch'`. This is as easy as it sounds. Such a property also has other uses: #771 introduces this property for the parliament corpora to set analyser properties. In light of the UB being interested in a broader range of languages, this may also be useful metadata to display in the corpora overview.

The issue with this is that for some corpora, the language varies between fields, or between documents. Possible solutions:

- For a corpus like DIOPTRA-L, the language varies between documents but is neatly stored in the `language` field. We could write the logic that we should get the language of a document from the `language` field if there is one, and otherwise from the corpus language property.
- For some corpora, such as the parliament corpora, _some_ of the fields are in the source language, and some are in English. We could add a boolean property to fields like `in_source_language`, that determines whether `lang` should be set or not.
- In a more complicated case, we might have documents that have multiple non-English languages. The logic suggested above does not account for that option, though I don't think we have any of those at the moment. For complete customisability and future-proofness, we could allow fields to have a `language` multifield, which will be given priority if present.
